module/data/actors/actor-kite.mjs

```diff
@@ -0,0 +1,99 @@
+import { CORIOLIS_TGD } from "../../config/config.mjs";
+import { DataHelper } from "../../helpers/data.mjs";
+import RollDialogWithConnectedActorAutomation from "../automations/roll-dialog-with-connected-actor.mjs";
+import RollDialogAutomation from "../automations/roll-dialog.mjs";
+import cgdVehicle from "./actor-vehicle.mjs";
+
+export default class cgdKite extends cgdVehicle {
+  static LOCALIZATION_PREFIXES = [
+    ...super.LOCALIZATION_PREFIXES,
+    'CORIOLIS_TGD.Actor.Kite',
+    'CORIOLIS_TGD.Item.Equipment',
+  ];
+
+  static defineSchema() {
+    const fields = foundry.data.fields;
+    const schema = super.defineSchema();
+
+    schema.bonus = new fields.NumberField({ ...DataHelper.requiredInteger, initial: 0, min: 0 });
+    schema.maxBonus = new fields.NumberField({ ...DataHelper.requiredInteger, initial: 0, min: 0 });
+    schema.range = new fields.NumberField({ ...DataHelper.requiredInteger, initial: 0, min: 0 });
+    schema.tech = new fields.SetField(new fields.StringField({ required: true, choices: CORIOLIS_TGD.Equipment.techChoices }), { initial: [CORIOLIS_TGD.Equipment.techConstants.ordinary] });
+    
+    return schema;
```

Comment:
It is not here, but a Kite should not have passengers. So, maybe we should delete that from the schema for Kite.

On the other side, Kites have a cost that is not on the list today.

Reply:
Addressed in [bcbbc36](https://github.com/hodpub/coriolis-tgd/pull/44/commits/bcbbc3661fd18f8270b20848777725b63b798628); remove/delete the passengers, add the cost.